appveyor.yml

```diff
@@ -15,3 +15,8 @@ test_script:
 - ps: build_tools\run_ci_db_test.ps1 -EnableRerun -Run db_test -Exclude DBTest.GroupCommitTest -Concurrency 10
 - ps: build_tools\run_ci_db_test.ps1 -Run env_test -Concurrency 1
 
+
+artifacts:
+  - path: build\Release
```

Comment:
Release is not currently built for CI builds. What is built is an optimized Debug version to make tests run faster as tests are not desgined to run in Release.